src/components/RefinementList.vue

```diff
@@ -121,6 +124,11 @@ export default {
       type: Boolean,
       default: false,
     },
+    searchablePlaceholder: {
+      default: 'Search here…',
```

Comment:
This is not part of the [RefinementList spec](https://instantsearch-css.netlify.com/widgets/refinement-list/) yet, I'm not sure we want a default placeholder here?

In InstantSearch.js 3 it's `"Search..."` for now but I would rather go without any.

Reply:
there will always be a placeholder since we use ais-search-box, we should probably spec it

also, go back to rest 👨‍⚕️